0_RootFS/gcc_sources.jl

```diff
@@ -175,6 +186,7 @@ function gcc_sources(gcc_version::VersionNumber, compiler_target::Platform; kwar
             v"11.1.0" => v"2.36",
             v"12.1.0" => v"2.38",
             v"13.2.0" => v"2.41",
+            v"14.2.0" => v"2.41",
```

Comment:
We should upgrade binutils, at least v2.42 (which was released few months before GCC 14), if not 2.43.1, latest version available.

Reply:
Done.